IanGRap/Squish
# Fix meta files

Contributors: Alfredo

Goal: Delete files that were misplaced and causing meta file weirdness. Added back text files used to make sure that git properly tracks Plugins/Editor/

Changes: 
* Deleted a Editor folder that was accidentally moved into the Materials Folder
* Deleted Plugin.meta~... files. These seemed to be a result of a merge but weren't being used
* Deleted an unknown file called _h origin fix-random-deaths_; Don't know why this file exited. I deleted it and nothing seemed to break. I know Unity doesn't use it so maybe it was a git thing? 
* Added empty.txt files to the Plugins/ folder and Plugins/Editor/ folder. These txt files ensure that the folder is not empty. This is necessary because git wont track empty folders

How to test: 
* Open up unity and visual studio while on this branch. There shouldn't be any changes that happen just because Unity is opened. 
* Run through the game normally. Everything should be working fine. We want to make sure that deleting the _h origin fix-random-deaths_ file didn't break anything. 
* Look inside of Plugins/Editor/ folder. There should only be empty.txt files in these folders (unless you are using a different editor than Visual Studio, then your editor might add changes here)
